src/GenerateTemplateFiles.ts

```diff
@@ -51,6 +51,9 @@ export default class GenerateTemplateFiles {
         const replacers: IReplacer[] = slots.map((str: string) => {
             const [slot, slotValue] = str.split('=');
 
+            const isValidReplacer: boolean = Boolean(slot) && str.includes('=');
+            CheckUtility.check(isValidReplacer, `${str} is not valid as a IReplacer`);
```

Comment:
@knona added validation for `IReplacer`. I don't valid `slotValue` because the user is allowed to pass an empty string for the `slotValue`.

Reply:
What i meant was that you should check if the number of arguments equals the size of the array of string replacers.
 Something like that : 
`replacers.length === selectedConfigItem.stringReplacers.length`

Reply:
And maybe you should check too is the slot is a valid replacer. I was doing that :

![image](https://user-images.githubusercontent.com/44003298/80915919-992fba80-8d55-11ea-95a5-7200baef049d.png)

Where "option" corresponds to "slot" in your code

Reply:
This is not when running the command line right? Because `selectedConfigItem.stringReplacers` are ignored when using the command line.

Reply:
@knona I am going to merge this PR into master and do a release. Please branch off of master and create another PR. That will help me understand the issue better.

Reply:
Yes it is. Indeed I used `selectedConfigItem.stringReplacers` to know what slots are needed to run the option.

Reply:
Therefore if an user define two string replacers in the configItem and define only one in the command line, I can inform him that one argument is missing..

Reply:
What if the user is only setup the ConfigItem only to be used with the command line. stringReplacers is optional at this time. I will have to think about if we should force the user to have the `stringReplacers` and command line arguments be the same. Do think they should be?

In my [readme](https://github.com/codeBelt/generate-template-files#command-line-usage) I say "When using the command line `stringReplacers` will be ignored, and the arguments pass in will be used."

Reply:
Exactly, I think they should be the same.

Reply:
> @knona I am going to merge this PR into master and do a release. Please branch off of master and create another PR. That will help me understand the issue better.

Sorry I didn't see this message. Do you want me to do an issue ?

Reply:
Sure, create an issue.